integration-tests/plugins/__tests__/cart/store/carts.spec.ts

```diff
@@ -77,13 +80,74 @@ describe("Store Carts API", () => {
         name: "Webshop",
       })
 
+      const [product] = await productModule.create([
+        {
+          title: "Test product",
+          variants: [
+            {
+              title: "Test variant",
+            },
+            {
+              title: "Test variant 2",
+            },
+          ],
+        },
+      ])
+
+      const priceSet = await pricingModule.create({
```

Comment:
I assume we can do `const [first, second] = create([first, second]`? At least it's supported like that in the other modules

Reply:
Yes, that is possible. Will update.